pkg/plugins/backendplugin/backend_plugin.go

```diff
@@ -98,6 +98,8 @@ func (p *BackendPlugin) start(ctx context.Context) error {
 			}
 		}
 	} else {
+		p.logger.Warn("Plugin uses a deprecated version of Grafana's backend plugin system which will be removed in a future release. " +
+			"Consider upgrade to a newer plugin version or reach out to the plugin repository/developer and request an upgrade. ")
```

Comment:
"Consider upgrading" or "Consider an upgrade". Should there be a space after the final period? That seems odd.

Reply:
```suggestion
			"Consider upgrading to a newer plugin version or reach out to the plugin repository/developer and request an upgrade.")
```

Reply:
I fixed it :smile: Thanks